disks-and-volumes.md

```diff
@@ -258,6 +258,47 @@ spec:
         claimName: mypvc
 ```
 
+### Ephemeral Volume
+
+An ephemeral volume is a local COW (copy on write) image that uses a network
+volume as a read-only backing store. With an ephemeral volume, the network
+backing store is never mutated. Instead all writes are stored on the ephemeral
+image which exists on local storage. KubeVirt dynamically generates the
+ephemeral images associated with a VM when the VM starts, and discards the
+ephemeral images when the VM stops.
+
+Ephemeral volumes are useful in any scenario where disk persistence across VM
+reboots is not desired.
```

Comment:
It will survive a reboot of a vm. We should make clear that it gets discarded once the VM reaches a a final state (succeeded, failed), but that it will survive soft reboots from within the vm.